.github/workflows/sync-code-in-readme.yml

```diff
@@ -0,0 +1,12 @@
+name: Embed code in README
```

Comment:
why are we adding this into repositories, we should only add those workflow that are relevant for given repository.